sources/web/datalab/polymer/modules/file-manager-factory/file-manager-factory.ts

```diff
@@ -22,7 +22,7 @@ enum FileManagerType {
 }
 
 interface FileManagerConfig {
-  className: new () => FileManager;
+  class: new () => FileManager;
```

Comment:
I'm surprised this compiles, I expected that class is a reserved word. Apparently it is not, but in my mind it kind of is. For clarity, I think it should be something else. But if you don't share that concern, you can go ahead and submit as is.

Reply:
Yea you can do that, but I mostly share your concern. Renamed to `typeClass`.